inc/search.class.php

```diff
@@ -88,15 +88,21 @@ class PluginFusioninventorySearch extends CommonDBTM {
    /**
     * Cloned Core function to display with our require.
     *
+    * !!! This function is never used anywhere in the plugin !!!
```

Comment:
It should be removed if not used (I do not see any usage); but it may be in plugins that depends on FI.

Reply:
I confirmed with @ddurieux that this function has been used but is now deprecated. I simple commented to raise an alert before removing 😉 but I will remove this code.

Reply:
You can also mark it as deprecated in the phpodc (using '@deprecated`) so this can be removed later. As you prefer ;)